web-test-runner.config.js

```diff
@@ -1,38 +1,13 @@
-import { playwrightLauncher } from '@web/test-runner-playwright';
+import { createConfig, getBrowsers } from '@brightspace-ui/testing';
 
-function getPattern(type) {
-	return `+(components|controllers|directives|helpers|mixins|templates)/**/*.${type}.js`;
-}
+const pattern = type => `+(components|controllers|directives|helpers|mixins|templates)/**/*.${type}.js`;
 
-export default {
-	files: getPattern('test'),
-	nodeResolve: true,
-	groups: [
-		{
-			name: 'aXe',
-			files: getPattern('axe'),
-			browsers: [
-				playwrightLauncher({
-					async createPage({ context }) {
-						const page = await context.newPage();
-						await page.emulateMedia({ reducedMotion: 'reduce' });
-						return page;
-					}
-				})
-			]
-		}
-	],
-	testFramework: {
-		config: {
-			ui: 'bdd',
-			timeout: '20000',
-		}
-	},
-	testRunnerHtml: testFramework =>
-		`<html lang="en">
-			<body>
-				<script src="./tools/resize-observer-test-error-handler.js"></script>
-				<script type="module" src="${testFramework}"></script>
-			</body>
-		</html>`
-};
+export default createConfig({
+	pattern,
+	vdiff: true,
```

Comment:
If we're planning to merge this right away, I think we want this off.

Reply:
All it does is add the group, which you'd have to explicitly target, and if we remove the `test:vdiff` script, which I think we should, it'll allow us to run it with `npx`. But we can turn it off here and just flip it on when we want to test it instead.

Reply:
Ok. I was figuring we'd still be maintaining some kind of WIP branch like my PoC one where we could toggle it on and do whatever else we wanted to, but we can sort that out later once we know more about how we're going to use/test this in core while still developing the vdiff library.

Reply:
Yeah we can do either or both. I'll turn these things off.